src/vuestic-theme/vuestic-components/vuestic-breadcrumbs/VuesticBreadcrumbs.vue

```diff
@@ -29,7 +29,19 @@
           return []
         }
       }
-    }
+    },
+    methods: {
```

Comment:
instead of such tricks it is better to pass configuration object to breadcrumbs as a property from the application.

As an example - Some of the breadcrumbs' routes must be disabled (e.g Home -> Forms -> Form Elements). Form elements must be disabled. Also root path (Home) must be also configurable